app/Commands/Dev/Core/Asm.hs

```diff
@@ -5,16 +5,15 @@ import Commands.Base
 import Commands.Dev.Core.Asm.Options
 import Juvix.Compiler.Asm qualified as Asm
 import Juvix.Compiler.Core qualified as Core
-import Juvix.Compiler.Core.Translation.Stripped.FromCore qualified as Stripped
 
-runCommand :: forall r a. (Members '[Embed IO, App] r, CanonicalProjection a CoreAsmOptions) => a -> Sem r ()
+runCommand :: forall r a. (Members '[Embed IO, App, TaggedLock] r, CanonicalProjection a CoreAsmOptions) => a -> Sem r ()
 runCommand opts = do
-  gopts <- askGlobalOptions
   inputFile :: Path Abs File <- fromAppPathFile sinputFile
+  ep <- getEntryPoint sinputFile
   s' <- readFile $ toFilePath inputFile
   tab <- getRight (mapLeft JuvixError (Core.runParserMain inputFile defaultModuleId mempty s'))
-  r <- runReader (project @GlobalOptions @Core.CoreOptions gopts) $ runError @JuvixError $ Core.toStripped' (Core.moduleFromInfoTable tab)
-  tab' <- Asm.fromCore . Stripped.fromCore . Core.computeCombinedInfoTable <$> getRight r
+  r <- runReader ep $ runError @JuvixError $ coreToAsm (Core.moduleFromInfoTable tab)
```

Comment:
Because of how ormolu handles multiline formatting of infix `.` and `$`, it is better to have "f . g $ x" rather than "f $ g $ x". This is a pattern that we try to follow in the codebase. When running effects, we usually run each effect in a separate line. E.g.
```
run 
 . runReader r
 . runError @JuvixError
 $ x
```

```suggestion
  r <- runReader ep . runError @JuvixError $ coreToAsm (Core.moduleFromInfoTable tab)
```

Reply:
done